sharktank/tests/models/flux/flux_test.py

```diff
@@ -242,6 +242,9 @@ def testCompareDevIreeF32AgainstHuggingFaceF32(self):
             reference_dtype=torch.float32, target_dtype=torch.float32, atol=1e-2
         )
 
+    @pytest.mark.skip(
+        reason="Segmentation fault during output comparison. See https://github.com/nod-ai/shark-ai/actions/runs/13704870816/job/38327614337?pr=1003"
```

Comment:
```suggestion
        reason="Segmentation fault during output comparison. See https://github.com/nod-ai/shark-ai/issues/1050"
```
Logs are only stored for a limited time, thus rather link to the issue (and post the relevant parts of the log there).